include/librealsense2/hpp/rs_options.hpp

```diff
@@ -5,9 +5,21 @@
 #define LIBREALSENSE_RS2_OPTIONS_HPP
 
 #include "rs_types.hpp"
+#include "../h/rs_types.h"
 
+#include <memory>
+
+// API struct, defined in general namespace
+struct rs2_options_list
+{
+    std::vector< rs2_option > list;
```

Comment:
This should not be declared here...

Reply:
Then where is the correct place for it?
`options_list` implementation need it.
Previously it was defined in `syntethic_sensor.h` I don't think that is the right place too...

Reply:
Just like `rs2_device_list` -- in rs.cpp

Reply:
As for the implementation needing it -- it does not. We should do the same thing as in `rs2_device_list`.

Reply:
Done